demo/vagrant/horizontal-app-scaling/jobs/haproxy.nomad

```diff
@@ -59,6 +72,15 @@ EOF
           interval = "10s"
           timeout  = "2s"
         }
+
+        check {
+          name = "host"
+          type     = "http"
+          address_mode = "driver"
```

Comment:
I believe we can omit this, and allow the driver to dictate the address_mode for the check.